src/main/java/seedu/address/model/person/DateOfBirth.java

```diff
@@ -60,13 +61,21 @@ public int compareTo(DateOfBirth d) {
     }
 
     /**
-     * Returns the the String representation of the DateOfBirth in format suitable for storage logging
+     * Returns the the String representation of the DateOfBirth in format suitable for storage logging.
      * @return String
      */
     public String toLogFormat() {
         return this.date.format(logFormatter);
     }
 
+    /**
+     * Returns the integer age in years of a person with this date of birth.
```

Comment:
```suggestion
     * Returns the integer age of a person.
```

Reply:
i thought it needed to be clearer to specify it is the age in years, but more general is fine as well i've changed it